content/legal/acceptable-use.md

```diff
@@ -0,0 +1,76 @@
+---
+title: "Acceptable Use"
+date: 2018-06-07T15:09:28-07:00
+menu:
+    legal:
+        weight: 2
+
+hero_title: "Acceptable Use Policy"
+url: "/acceptable-use"
```

Comment:
As a subsequent exercise, I wonder if we should audit all the uses of `url:` and consider just placing these files where they _should_ be, e.g. instead of having this in `/content/legal/acceptable-use.md` with a URL that makes it `/acceptable-use`, delete the `url:` front matter and just place the file directly in `/content/acceptable-use.md`